src/htmltojsx.js

```diff
@@ -468,6 +481,12 @@ StyleParser.prototype = {
       var key = style.substr(0, firstColon);
       var value = style.substr(firstColon + 1).trim();
       if (key !== '') {
+        // Lowercase style name if not vendor specific
+        // TODO better vendor prefix handling
+        if(key[0] != '-') {
```

Comment:
Use triple equals for string equality, `!==`.

Can you add a test for the browser prefix case?